dbt/models/overview.md

```diff
@@ -0,0 +1,18 @@
+{% docs __overview__ %}
+# Cook County Assessor Data Department Catalog
```

Comment:
The text here is mostly placeholder text so far, but I'm open to edits nonetheless!